iina/pt-BR.lproj/Localizable.strings

```diff
@@ -1,342 +1,219 @@
-/* 
-  String_ui.strings
-  iina
+/** Generated by IINA Translation Utility */
 
-  Created by lhc on 3/1/2017.
-  Copyright © 2017 lhc. All rights reserved.
-*/
-
-// Common
-"general.on" = "Ligar";
-"general.off" = "Desligar";
+"alert.add_filter.message" = "Entre com uma string no formato de comandos vf ou af do mpv.";
+"alert.add_filter.title" = "Novo Filtro";
+"alert.add_url.message" = "Por favor insira a URL";
+"alert.add_url.title" = "Adicionar URL";
+"alert.add_watch.message" = "Entre com um nome de propriedade mpv válida.";
+"alert.add_watch.title" = "Adicionar propriedade de exibição";
+"alert.assrt_register" = "Devido à quota limitada da API, recomendamos vivamente que utilize o seu próprio token da API. O token padrão é apenas para uso de demonstração e você pode encontrar um erro de rede.";
+"alert.assrt_register.register" = "Registrar Agora";
+"alert.assrt_register.try" = "Tente primeiro";
+"alert.assrt_token_invalid" = "Seu token da API do Assrt parece ser inválido.";
+"alert.assrt_token_prompt.message" = "Por favor, registre-se na página da web aberta. Obtenha seu token na página inicial do usuário e insira-o abaixo:";
+"alert.assrt_token_prompt.title" = "Insira seu token de API";
+"alert.beta_channel.message" = "IINA v1.0 está chegando! Gostaria de nos ajudar a testar versões beta?\n\nVocê pode alterar esse comportamento a qualquer momento em Preferências - Geral.";
+"alert.beta_channel.title" = "Receba atualizações para as versões beta";
+"alert.choose_media_file.title" = "Escolha o arquivo de mídia";
+"alert.clear_cache.message" = "Tem certeza que deseja limpar o cache de miniaturas?";
+"alert.clear_cache.title" = "Limpar cache de miniaturas";
+"alert.clear_history.message" = "Tem certeza que deseja excluir todo o histórico de reprodução?";
+"alert.clear_history.title" = "Limpar histórico de reprodução";
+"alert.clear_watch_later.message" = "Tem certeza que deseja excluir o progresso de reprodução (dados de \"ver mais tarde\")?";
+"alert.clear_watch_later.title" = "Limpar progresso de reprodução";
+"alert.config.cannot_create" = "Não consigo criar arquivo de configuração!";
```

Comment:
The source string is
```
Cannot create config file!
%@
```
Please append \n%@ to the localized string.

Reply:
Fixed